src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -163,10 +163,18 @@ public void logout() {
    * global context directory.
    */
   public void fetchPetSaCredentials() {
+    // if the cloud context is undefined, then something went wrong during workspace creation
+    // just log an error here instead of throwing an exception, so that the workspace load will
+    // will succeed and the user can delete the corrupted workspace
+    String googleProjectId = Context.requireWorkspace().getGoogleProjectId();
```

Comment:
aside: Are workspaces without cloud contexts always erroneous? Our model allows them AFAIK, and at least one test script uses this approach to save time, as referenced resources don't require cloud contexts.

Reply:
For the CLI, yes they are. `terra workspace create` creates a workspace and a Google cloud context. This may need to change in the multi-cloud future, but for now, the only way you can get a workspace w/o a cloud context from the CLI is if there's a failure.